lib/platform/common.ts

```diff
@@ -206,9 +206,14 @@ export interface Platform {
     branchName: string,
     context: string
   ): Promise<BranchStatus | null>;
-  ensureCommentRemoval(
-    ensureCommentRemoval: EnsureCommentRemovalConfig
-  ): Promise<void>;
+  ensureCommentRemoval(ensureCommentRemoval: {
```

Comment:
I would like to improve the types here, will send a commit later.

Reply:
a commit to master after it's merged, or to this PR?

Reply:
done